cfme/tests/automate/custom_button/test_buttons.py

```diff
@@ -24,28 +27,14 @@
 
 
 @pytest.fixture(scope="module")
-def buttongroup(appliance):
-    def _buttongroup(object_type):
-        collection = appliance.collections.button_groups
-        button_gp = collection.create(
-            text=fauxfactory.gen_alphanumeric(),
-            hover=fauxfactory.gen_alphanumeric(),
-            type=getattr(collection, object_type),
-        )
-        return button_gp
-
-    return _buttongroup
-
-
-@pytest.fixture(params=OBJ_TYPE, ids=[obj.capitalize() for obj in OBJ_TYPE], scope="module")
-def button_group(appliance, request):
+def button_group(appliance, obj_type):
     collection = appliance.collections.button_groups
     button_gp = collection.create(
         text=fauxfactory.gen_alphanumeric(start="grp_"),
         hover=fauxfactory.gen_alphanumeric(start="hover_"),
-        type=getattr(collection, request.param),
+        type=getattr(collection, obj_type),
     )
-    yield button_gp, request.param
+    yield button_gp
```

Comment:
when you shorten names, please pay attention to their readability. I'd shorten group to grp which explains contents better.